docs/install.md

```diff
@@ -1,38 +1,69 @@
 Install
 =======
 
+This document describes how to install the Machine code for local development, and demonstrates two ways to use it: running a single source and running a complete batch set. If you’re editing a lot of sources and want to do it quickly without waiting for a remote Github-based continuous integration service, you may want to use run single sources locally. If you're working on the queuing and job control portions of Machine code, you may want to run complete batch sets on test data.
+
 Local Development
 -----------------
 
 You can edit a local copy of OpenAddresses code with working tests by installing
-everything onto an Ubuntu Linux virtual machine. We recommend using a dedicated
-virtual machine for development; our Chef script installs servers, packages, and
-other configuration that may disrupt other uses of an Ubuntu machine. This
-process should take about 20 minutes depending on download speed, or you can
-skip the first four steps by using a pre-built Ubuntu 14.04 image on Amazon EC2.
+everything onto a [Vagrant virtual machine](https://www.vagrantup.com). We
+recommend using a dedicated virtual machine for development; our Chef script
+installs servers, packages, and other configuration that may disrupt other uses
+of an Ubuntu machine. This process should take about 10-20 minutes depending on
+download speed.
```

Comment:
I was confused at first about whether I needed to start a VirtualBox Ubuntu instance in preparation for using Vagrant.  It's my understanding that Vagrant initializes a VirtualBox ubuntu instance.  Is that correct?